src/Components/Publishing/Display/DisplayAd.tsx

```diff
@@ -28,10 +32,27 @@ export interface DisplayAdContainerProps extends FlexProps {
 GPT.syncCorrelator(true)
 
 export const DisplayAd: SFC<DisplayAdProps> = props => {
-  const { adDimension, adUnit, targetingData, ...otherProps } = props
+  const {
+    adDimension,
+    adUnit,
+    targetingData,
+    articleSlug,
+    ...otherProps
+  } = props
   const [width, height] = adDimension.split("x").map(a => parseInt(a))
   const [isAdEmpty, setAdEmpty] = useState(null)
   const isMobileLeaderboardAd = is300x50AdUnit(adDimension)
+  const { trackEvent } = useTracking()
+
+  const onClickAd = () => {
+    trackEvent({
+      context_page: AnalyticsSchema.PageName.ArticlePage,
+      context_module: AnalyticsSchema.ContextModule.AdServer,
+      context_page_owner_id: props.targetingData.post_id,
+      context_page_owner_slug: props.articleSlug,
+      context_page_owner_type: AnalyticsSchema.OwnerType.Article,
+    })
+  }
```

Comment:
For the [click event](https://github.com/artsy/reaction/blob/e92baf0ba0cb3aa4d25f448969d6ec926f24fd73/src/Apps/Collect2/Routes/Collection/Components/CollectionsHubRails/OtherCollectionsRail/OtherCollectionEntity.tsx#L25), I think we'd want to `action_type: Schema.ActionType.Click` here.